docs/evm/how-it-works.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences How EVM on Flow Works
```

Comment:
Would update this as well to: How EVM on Flow Works

Reply:
fixed